install.sql

```diff
@@ -11,7 +11,10 @@ CREATE OR REPLACE FUNCTION is_claims_admin() RETURNS "bool"
       --------------------------------------------
       IF extract(epoch from now()) > coalesce((current_setting('request.jwt.claims', true)::jsonb)->>'exp', '0')::numeric THEN
         return false; -- jwt expired
-      END IF; 
+      END IF;
```

Comment:
I would add a comment similar to the original comment explaining what this logic does and that one can disable it if we don't want the service role to override this claim check.